README.md

```
@@ -1,9 +1,190 @@
+
+Environments are independent from each-other and cannot be cross-authenticated, each token is linked to an environment, and cannot be used to cross-authenticate into multiple environments.
+
+When `--domain` is not specified, the last logged-in business becomes `current`.
+
+## Terminal autocomplete
+
+Kinde CLI comes with terminal autocompletion support for `bash`, `fish`, `powershell` and `zsh`
+
+You can see instruction for your shell/OS combination by executing the following
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)